src/NumericalAlgorithms/Interpolation/IrregularInterpolant.cpp

```diff
@@ -4,15 +4,41 @@
 #include "IrregularInterpolant.hpp"
 
 #include <array>
+#include <vector>
 
 #include "DataStructures/DataVector.hpp"
 #include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/LogicalCoordinates.hpp"
 #include "NumericalAlgorithms/Spectral/Mesh.hpp"
 #include "NumericalAlgorithms/Spectral/Spectral.hpp"
 // IWYU pragma: no_forward_declare Tensor
 
 namespace {
 
+// Just linear for now, can be extended to higher order...
+// Future optimization?  use Blaze sparse matrices
```

Comment:
could you expand a bit on this optimization note? Something like "Future optimization: it might be more efficient to use Blaze's sparse matrices since the interpolation matrix is mostly empty"